src/pytrajplot/generate_pdf.py

```diff
@@ -595,6 +597,10 @@ def generate_pdf(
                     "lat"
                 ] = trajectory_df["lat"][first_row:next_first_row]
 
+                # TODO: FutureWarning: The behavior of array concatenation with empty entries is deprecated.
```

Comment:
Maybe you want to pack this into an issue rather than having a comment within the code. You can more easily keep track of it.

Reply:
Good point, changed as suggested